.golangci.yml

```diff
@@ -1,38 +1,26 @@
 # golangci.com configuration
 # https://github.com/golangci/golangci/wiki/Configuration
-service:
-  golangci-lint-version: 1.57.2 # use the fixed version to not introduce new linters unexpectedly
```

Comment:
Do we really want to do this? I've found it useful to have commits where the golangci version is bumped and then all of the corresponding new linter fixes get added as a self-contained change. It makes the process more deliberate.